errbot/plugin_manager.py

```diff
@@ -384,6 +385,22 @@ def update_plugin_places(self, path_list, extra_plugin_dir, autoinstall_deps=Tru
                        for pluginfo in loaded_plugins if pluginfo.error is not None})
         return errors
 
+    def get_all_active_plugin_objects_ordered(self):
+        all_plugins = []
+        for name in self.plugins_callback_order:
+            # None is a placeholder for any plugin not having a defined order
+            if name is None:
+                all_plugins += [
```

Comment:
We need a default behavior with no "None" otherwise we gonna skip all the other plugins.